packages/extension/src/lib/extensionScripts.ts

```diff
@@ -96,37 +103,12 @@ export const getHostPermission = (): Promise<boolean> =>
     origins: HOST_PERMISSIONS,
   });
 
-export const getHostPermissionAndRegister = async (): Promise<void> => {
+export const getHostPermissionAndRegister = async (): Promise<boolean> => {
```

Comment:
Intentional we now expect a return here?

Reply:
yes, we want to know in case the user clicked `deny` on this pop-up (which would result in a `false` returned by the fn call`